modules/solid_mechanics/doc/content/source/materials/MultiParameterPlasticityStressUpdate.md

```diff
@@ -0,0 +1,219 @@
+# MultiParameterPlasticityStressUpdate
```

Comment:
that's weird. any page in the folder appears, even if the object did not exist

Reply:
ok this isd normal then.
It s a base class it s not part of the syntax

however it is listed in the source list, because that lists all the existing pages